docs/changes/table/17184.api.rst

```diff
@@ -0,0 +1,3 @@
+``Table.pformat``'s default behavior was changed to include all rows and
+columns, instead of truncating outputs to fit the current terminal.
+``Table.pformat_all`` now being redundant, it is pending deprecation.
```

Comment:
```suggestion
The default behavior of ``Table.pformat`` was changed to include all rows and columns
instead of truncating the outputs to fit the current terminal.  The new default
keyword arguments ``max_width=-1`` and ``max_lines=-1`` now match those in
``Table.pformat_all``. Since the ``Table.pformat_all`` method is now redundant, it is
pending deprecation. Similarly, the default behavior of ``Column.pformat`` was changed
to include all rows instead of truncating the outputs to fit the current terminal.
```